batsat/src/lib.rs

```diff
@@ -0,0 +1,125 @@
+//! # rustsat-batsat - Interface to the `BatSat` SAT Solver for `RustSAT`
+//!
+//! Interface to the [BatSat](https://github.com/c-cube/batsat) incremental SAT-Solver to be used with the [RustSAT](https://github.com/chrjabs/rustsat) library.
+//!
+//! `BatSat` is fully implemented in Rust which has advantages in restricted compilation scenarios like WebAssembly.
+//!
+//! # `BatSat` Version
+//!
+//! The version of `BatSat` in this crate is Version 0.5.0.
```

Comment:
```suggestion
//! # rustsat-batsat - Interface to the BatSat SAT Solver for RustSAT
//!
//! Interface to the [BatSat](https://github.com/c-cube/batsat) incremental SAT-Solver to be used with the [RustSAT](https://github.com/chrjabs/rustsat) library.
//!
//! BatSat is fully implemented in Rust which has advantages in restricted compilation scenarios like WebAssembly.
//!
//! # BatSat Version
//!
//! The version of BatSat in this crate is Version 0.5.0.
```
Here's something I forgot to mention. Since these are proper names, rather than setting them as code, like Clippy suggests, I put them on an ignore list. RustSAT should be there already, so clippy actually shouldn't warn about that one. BatSat should be added to the list of ignored names in `clippy.toml`.